cmake/onnxruntime_hosting.cmake

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+if(NOT DEFINED CMAKE_TOOLCHAIN_FILE)
+    set(CMAKE_TOOLCHAIN_FILE "${ONNXRUNTIME_ROOT}/cmake/external/vcpkg/scripts/buildsystems/vcpkg.cmake" CACHE STRING "")
+endif()
+
+project(onnxruntime_hosting)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
this gets set at the start of CMakeLists.txt, so it might be redundant.

Reply:
Fixed.